code/modules/mob/living/simple_mob/subtypes/vore/mobs_monsters/clowns/bigclowns.dm

```diff
@@ -0,0 +1,701 @@
+/mob/living/simple_mob/mobs_monsters/clowns/big/normal
+	name = "Clown"
+	desc = "A regular, every tuesday Clown."
+	tt_desc = "E Homo sapiens corydon" //this is a clown
+	icon = 'icons/mob/mobs_monsters/newclowns.dmi'
+	icon_state = "c_normal"
+	icon_living = "c_normal"
+	icon_dead = "clown_dead"
+	icon_gib = "generic_gib"
+	vis_height = 32
+
+	faction = "clown"
+
+	loot_list = list(/obj/item/weapon/bikehorn = 100)
+
+	response_help = "pokes"
+	response_disarm = "gently pushes aside"
+	response_harm = "hits"
+
+	say_list_type = /datum/say_list/clown
+
+/datum/say_list/clown
+	speak = list("HONK", "Honk!", "Henk!")
+	emote_see = list("honks")
+
+
+/mob/living/simple_mob/mobs_monsters/clowns/big/honkmunculus
+	name = "A Clown?"
+	desc = "That clown has some interesting proportions."
+	tt_desc = "E Homo sapiens corydon horrificus" //this is a redspace clown
+	icon = 'icons/mob/mobs_monsters/TGClowns.dmi'
+	icon_state = "honkmunculus"
+	icon_living = "honkmunculus"
+	icon_dead = "honkmunculus_dead"
+	icon_gib = "generic_gib"
+	vis_height = 64
+
+	old_x = -16
+	old_y = 0
+	default_pixel_x = -16
+	pixel_x = -16
+	pixel_y = 0
+
+	faction = "clown"
+
+	loot_list = list(/obj/item/weapon/bikehorn = 100)
+
+	response_help = "pokes"
+	response_disarm = "gently pushes aside"
+	response_harm = "hits"
+
+	say_list_type = /datum/say_list/clown
+
+/datum/say_list/clown
+	speak = list("HONK", "Honk!", "Henk!")
+	emote_see = list("honks")
+
```

Comment:
You define this same thing several times. There's no need to.

Reply:
faction = "clown"

	loot_list = list(/obj/item/weapon/bikehorn = 100)

	response_help = "pokes"
	response_disarm = "gently pushes aside"
	response_harm = "hits"

	say_list_type = /datum/say_list/clown

/datum/say_list/clown
	speak = list("HONK", "Honk!", "Henk!")
	emote_see = list("honks")

All of this could be removed actually.

Reply:
Yap tru

Reply:
Donezo! Got them removed!